README.md

```diff
@@ -16,6 +16,31 @@ This chart demonstrates 3 scenarios:
 2. `logs-by-label` collects all logs for pods with the label `rancher.logging.example/app: log-generator`
 3. `namespace-logs` collects all logs for the namespace this chart is deployed in.
 
+#### Resource Requests and Limits

```

Comment:
We should move this into the `rancher-logging-example` readme. it does not apply  to the log generator or simple application.

Reply:
I think the `####` puts it in there? I can make it more obvious. Maybe we can use **bold** instead.

Reply:
nvm I can't count

Reply:
All good haha